# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../clockspeed/0003-include-time.h.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From 17b43b9682da7098a4325d6bce4eb9e882f6e8b9 Mon Sep 17 00:00:00 2001
From: Christian Wiese <christian.wiese@securepoint.de>
Date: Thu, 29 Sep 2011 15:12:51 +0200
Subject: [PATCH 3/6] clockview: fixed to include time.h for localtime function

---
 clockview.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/clockview.c b/clockview.c
index 1e23519..74ace31 100644
--- a/clockview.c
+++ b/clockview.c
@@ -1,5 +1,6 @@
 #include <sys/types.h>
 #include <sys/time.h>
+#include <time.h> /* localtime */
 #include "substdio.h"
 #include "readwrite.h"
 #include "strerr.h"
-- 
1.6.6.2

